lib/findbugs_translate_checkstyle_format/translate.rb

```diff
@@ -50,5 +45,16 @@ def fqcn_to_path(fqcn, xml)
         src.index(path) != nil
       end
     end
+
+    def set_dummy(xml, checkstyle)
+      dummy_src_dir = xml['BugCollection']['Project']['SrcDir']
+      dummy_src_dir = dummy_src_dir.first if dummy_src_dir.is_a?(Array)
+
+      checkstyle.add_element("file", {
+        'name' => dummy_src_dir
+        })
```

Comment:
__[rubocop]__
* Indent the right brace the same as the first position after the preceding left parenthesis.